src/x11common.zig

```diff
@@ -7,7 +7,7 @@ pub const SocketReader = std.io.Reader(std.os.socket_t, std.os.RecvFromError, re
 pub fn send(sock: std.os.socket_t, data: []const u8) !void {
```

Comment:
All of the changes in this file are just the latest copied from the `zigx` repo: https://github.com/marler8997/zigx/blob/a313162bdac3182c7fc38a31dfa952c68ff861f9/common.zig